pkg/kubernetes/client_test.go

```diff
@@ -553,3 +556,50 @@ var _ = Describe("Test Namespace KubeController Methods", func() {
 	})
 
 })
+
+func TestGetEndpoint(t *testing.T) {
```

Comment:
Test case looks good to me. 

I suggest, we follow the testing style in the code above (using ginkgo) rather than assertion style.

Reply:
@allenlsy the recommendation is to use Go testing lib instead of Ginkgo for newer tests (ref: https://github.com/openservicemesh/osm/issues/1704)

Reply:
Thanks for the clarification.